src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -79,12 +79,12 @@ internal async Task ProcessHttpInAndOutInvocationRequest(ScriptInvocationContext
                 Outputs = new Dictionary<string, object>()
             };
 
-            (string name, DataType type, object request) input = scriptInvocationContext.Inputs.First();
+            (string Name, DataType Type, object Request) input = scriptInvocationContext.Inputs.First();
```

Comment:
Do you know why we have both the deconstructor and the `input` variable? Typically, for deconstructors you only have that and then refer to them directly:

``` CSharp
(string name, DataType type, object request) = scriptInvocationContext.Inputs.First();
HttpRequest httpRequest = request as HttpRequest;
....
```

Alternatively, you use `var` and let the signature naming take hold:

``` CSharp
var input = scriptInvocationContext.Inputs.First();
HttpRequest httpRequest = input.Request as HttpRequest;
```

Reply:
Not sure -- and in fact, in this case, it's odd b/c the third element in the tuple doesn't actually match the name of the one defined in the method (`Request` vs `Val`). I'm going to change this to `var` to make this more straightforward.